js/theme/view.js

```diff
@@ -239,6 +250,45 @@ function attachSlideshow() {
   }
 }
 
+function attachPermalink() {
+  const permalinks = document.querySelectorAll('.lb-post-permalink a');
+
+  permalinks.forEach((link) => {
+    link.href = permalink.getUrl(link.id);
+  });
+
+  return permalink._changedSort;
```

Comment:
from the name of the method is not clear that a sort status is returned.
Please rename it or add the status sort in another method.